packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -776,7 +796,7 @@ class EditableText extends StatefulWidget {
 class EditableTextState extends State<EditableText> with AutomaticKeepAliveClientMixin<EditableText>, WidgetsBindingObserver, TickerProviderStateMixin<EditableText> implements TextInputClient, TextSelectionDelegate {
   Timer _cursorTimer;
   bool _targetCursorVisibility = false;
-  final ValueNotifier<bool> _cursorVisibilityNotifier = ValueNotifier<bool>(true);
+  ValueNotifier<bool> _cursorVisibilityNotifier;
```

Comment:
I'm forgetting if this is allowed because it's not static:

```
final ValueNotifier<bool> _cursorVisibilityNotifier = ValueNotifier<bool>(widget.showCursor)
```

If not, we should still make this final and just set it in `initState`

Reply:
it is not allow, but i will update it to final